workflow-steps/cache/output/post.js

```diff
@@ -5973,15 +5982,14 @@ if (!!cacheWasHit) {
       baseUrl: "http://127.0.0.1:9000"
     })
   );
-  const currentBranch = process.env.NX_BRANCH;
   if (!input_key || !input_paths) {
     throw new Error("No cache restore key or paths provided.");
   }
   const key = hashKey(input_key);
   const paths = input_paths.split("\n").filter((p) => p);
-  cacheClient.store(
+  cacheClient.storeV2(
     new StoreRequest({
-      key: `${currentBranch}-${key}`,
+      key,
```

Comment:
@Cammisuli Is `key` correct? I'm seeing store v2 keeps failing to find cache from `base-branch`.

Reply:
My bad. I should have set `base_branch`.